config/superglue-bow.conf

```diff
@@ -0,0 +1,86 @@
+// Config settings used for SuperGLUE CBoW baseline experiments. 
+
+// This imports the defaults, which can be overridden below.
+include "defaults.conf"
+exp_name = "superglue-bow"
+
+// Data and preprocessing settings
+max_seq_len = 256
+tokenizer = MosesTokenizer
+
+// Model settings
+sent_enc = "bow"
+word_embs = glove
+skip_embs = 0
+sep_embs_for_skip = 0
+elmo = 0
+classifier = log_reg // following BERT paper
+pair_attn = 0 // shouldn't be needed but JIC
+s2s = {
+    attention = none
+}
+transfer_paradigm = finetune
+
+// Training settings
+dropout = 0.1
+optimizer = adam
+batch_size = 16
+max_epochs = 10
+lr = .00001
```

Comment:
FWIW, this seems pretty low for non-BERT.

Reply:
:/